CRM/Contribute/Form/Contribution.php

```diff
@@ -491,7 +491,7 @@ public function setDefaultValues() {
   public function buildQuickForm() {
     
     // FIXME: This probably needs to be done in preprocess
```

Comment:
@pradpnayak any ideas how I can get this working in preProcess instead of buildForm?